src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString.split(';').reduce((prev, item) => {
+    const trimmedItem = item.trim();
+
+    if (trimmedItem) {
+      const [key, value] = trimmedItem.split(':').map((part) => part.trim());
+
+      if (key && value) {
+        prev[key] = value;
+      }
+    }
+
+    return prev;
+  }, {});
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation. This makes the code much more readable. Align the methods `.split`, `.reduce`, and `.map` correctly.